README.md

```diff
@@ -51,21 +52,42 @@ class { 'lastpass': }
 
 Once installed, the lpass command is available as documented on the [user guide](https://lastpass.github.io/lastpass-cli/lpass.1.html).
 
+### Setting configuration options
+
+Lastpass is configured with environment variables. You can optionally set them in $LASTPASS_HOME/env.  
+A defined type is provided to manage these settings.
+
+```puppet
+lastpass::config { 'LPASS_AGENT_TIMEOUT':
+  ensure => present,
+  value  => 3600,
+}
+
+lastpass::config {
+  'agent_timeout':  value => 3600;
+  'auto_sync_time': value => 5;
+}
+
+lastpass::config { 'LPASS_AGENT_TIMEOUT':
+  ensure => absent,
+}
+```
+
 ### Automated login
 
-The module provides two helper scripts to facilitate automated login for unattended systems. These scripts require the LastPass username be placed in $LASTPASS_HOME/user and the password in $LASTPASS_HOME/pw. The module will create these if the home, username and password parameters are provided, or they can be created manually for the desired user. The default value of $LASTPASS_HOME is '$HOME/.lpass'.
+The module provides two helper scripts to facilitate automated login for unattended systems. These scripts require the LastPass username and password be stored in $LASTPASS_HOME/login. The module will create this if the username and password parameters are provided, or this can be created manually for the desired user. The default value of $LASTPASS_HOME is '$HOME/.lpass'.
 
-The lpasslogin script will read the $LASTPASS_HOME/user file and run 'lpass login $USERNAME'. If $LASTPASS_HOME/pw is present, the profile script installed by the module will set $LASTPASS_ASKPASS to the lpasspw script, which reads $LASTPASS_HOME/pw and prints it to STDOUT in response to the CLI.
+The lpasslogin script will read the $LASTPASS_HOME/login file and run 'lpass login $LPASS_USERNAME'. If $LASTPASS_HOME/login is present, the profile script installed by the module will set $LASTPASS_ASKPASS to the lpasspw script, which reads $LASTPASS_HOME/login and prints $LASTPASS_PASSWORD it to STDOUT in response to the CLI.
```

Comment:
fixed. Also added an example of the $LPASS_HOME/login file and fixed several other env var mistakes.